src/plugins/lua/known_senders.lua

```diff
@@ -197,6 +250,93 @@ local function known_senders_callback(task)
   end
 end
 
+local function verify_local_replies_set(task)
+  local replies_sender = task:get_reply_sender()
+  if not replies_sender then
+    lua_util.debugm(N, task, 'Could not get sender')
+    return nil
+  end
+
+  local replies_recipients = task:get_recipients('mime')
+
+  local replies_sender_string = lua_util.maybe_obfuscate_string(tostring(replies_sender), settings, settings.sender_prefix)
+  local replies_sender_key = make_key_replies(replies_sender_string:lower(), 8)
+
+  local function redis_zscore_script_cb(err, data)
+    if err ~= nil then
+      rspamd_logger.errx(task, 'Could not verify %s local replies set %s', replies_sender_key, err)
+    end
+    if data[1] ~= 1 then
+      rspamd_logger.infox(task, 'Recipients was not verified')
+    else
+      rspamd_logger.infox(task, 'Recipients was verified')
```

Comment:
Could be moved out of the conditional to avoid duplication or removed if this is misplaced.

Reply:
I don't quite understand what the problem is with this `if`. Could you please explain what you mean by duplication?

Reply:
I was reading the same log message on both of these lines somehow ....